packages/node_modules/@webex/plugin-meetings/src/meeting-info/utilv2.js

```diff
@@ -264,4 +264,40 @@ MeetingInfoUtil.getRequestBody = (options) => {
   return body;
 };
 
+MeetingInfoUtil.getWebexSite = (uri) => {
+  const exceptedDomains = ['meet.webex.com', 'meetup.webex.com', 'ciscospark.com'];
+  const site = uri?.match(/.+@(.+\..+\.*.)$/)?.[1];
```

Comment:
One of these lines should change.  Right now, my regex only matches URIs with 2 periods (e.g. cisco.webex.com) but the exceptedDomains I pulled from SCF includes 'ciscospark.com'.  Do we expect to ever see a domain without the cisco/www/etc?  I should either remove ciscospark.com or change the regex to match similar URIs.

Reply:
This function is pretty limited in that it only handles URIs that look like '10019857020@convergedats.webex.com'. Will any sipUri's have a different format?

Reply:
This is just optimization if we cannot match it appapi will redirect no issues

But ciscospark.com is not a webex site so we should neglect it 

Can you show where do you get ciscospark.com from

Reply:
https://sqbu-github.cisco.com/WebExSquared/spark-client-framework/blob/243e66ada181cd4ab94fae5b4053f4ce04997127/spark-client-framework/Services/WebexMeetingService/WebexMeetingService.cpp#L795

Reply:
i think its going to go away. we wont need it